blocks/core/ff_module/ff_module-profile-response-button/ff_module-profile-response-button.md

```diff
@@ -1,9 +1,32 @@
 <div data-ff_module-profile-response-button=""/>
 
-# Here be a profile-response button
+# Profile response button
 
 ## Markup 
 ```
 <div data-ff_module-profile-response-button=""/>
 ```
+### Props
 
+- `onSelect`: Function
+- `uiState`: String; css defined for `is-selected`, `is-updated` (default), `is-reviewed` in Melody
```

Comment:
As far as I understand, it's distinct from the state of the events themselves. It just means you've looked at the event stream for that recipient. As I said, the names might not be ideal. It might not need to be persisted, that was just my instinct - my assumption is that it's like the read status on an email. Looking at the design and talking with Rachel, the ones that have all blue text are 'unread', the ones that have grey text have been 'read', the selected ones are blue background.